extensions/blocks/podcast-player/podcast-player.php

```diff
@@ -224,6 +225,19 @@ function get_colors( $name, $attrs, $property ) {
 	return $colors;
 }
 
+function get_css_vars( $attrs ) {
+	$colors_name = array( 'primary', 'secondary', 'background' );
+
+	$inline_style = '';
+	foreach ( $colors_name as $color ) {
+		$hex_color = 'hex' . ucfirst( $color ) . 'Color';
+		if ( isset( $attrs[ $hex_color ] ) ) {
```

Comment:
`isset()` only checks if that key exists, the value could still be `null` or `''`, etc. 
`! empty()` will check if the value exists and is truthy, probably a better option here.